backend/src/config.ts

```diff
@@ -16,14 +16,24 @@ import { getApi } from './utils/connection'
 const envPath = path.resolve(__dirname, '../..', '.env')
 dotenv.config({ path: envPath })
 
+// Blockchain address with a default of Peregrine the testnet
 export const WSS_ADDRESS = process.env.WSS_ADDRESS ?? 'wss://peregrine.kilt.io'
-export const BACKEND_PORT = process.env.BACKEND_PORT ?? 2525
+export const BACKEND_PORT = process.env.BACKEND_PORT ?? 3000
 export const DAPP_ACCOUNT_MNEMONIC = loadEnv('DAPP_ACCOUNT_MNEMONIC')
 export const DAPP_DID_MNEMONIC = loadEnv('DAPP_DID_MNEMONIC')
 export const DAPP_DID_URI = loadEnv('DAPP_DID_URI') as Kilt.DidUri
 export const DAPP_NAME = process.env.DAPP_NAME ?? 'Web3-Login-Demo'
 export const JWT_SIGNER_SECRET = loadEnv('JWT_SIGNER_SECRET')
 
+// CType information
+export const CTYPE_HASH =
+  loadEnv('CTYPE_HASH') ||
+  '0x3291bb126e33b4862d421bfaa1d2f272e6cdfc4f96658988fbcffea8914bd9ac'
+export const TRUSTED_ATTESTERS =
+  loadEnv('TRUSTED_ATTESTERS') ||
+  'did:kilt:4pehddkhEanexVTTzWAtrrfo2R7xPnePpuiJLC7shQU894aY'
```

Comment:
Would be nice to move the hash and did to constants and describe what they are. (which attester, which ctype)